web/src/components/molecules/Dashboard/ProjectList.tsx

```diff
@@ -29,15 +32,15 @@ const ProjectList: React.FC<Props> = ({
         </Content>
       ) : (
         <EmptyListWrapper>
-          <Title>No Projects Yet</Title>
+          <Title>{t("No Projects Yet")}</Title>
           <Suggestion>
-            Create a new project{" "}
+            {t("Create a new project")}{" "}
             <Button onClick={handleProjectModalOpen} type="primary" icon={<Icon icon="plus" />}>
-              New Project
+              {t("New Project")}
             </Button>
           </Suggestion>
           <Suggestion>
-            Or read <a href="">how to use Re:Earth CMS</a> first
+            {t("Or read")} <a href="">{t("how to use Re:Earth CMS")}</a> {t("first")}
```

Comment:
Using react-router-dom's Link component is much nicer I have found and avoids any issues if we use other components from react-router-dom

Reply:
@KaWaite link here should redirect to an outer page (re:earth CMS guideline)